src/components/NurseryWithdrawals/WithdrawalLogRenderer.tsx

```diff
@@ -20,93 +20,83 @@ const useStyles = makeStyles((theme: Theme) => ({
+        showAllStyle={{ padding: theme.spacing(2), fontSize: 14 }}
+        listSeparator={strings.LIST_SEPARATOR}
+        moreSeparator={strings.TRUNCATED_TEXT_MORE_SEPARATOR}
+        moreText={strings.TRUNCATED_TEXT_MORE_LINK}
+      />
+    );
+  };
 
-    if (column.key === 'speciesScientificNames') {
-      return <CellRenderer index={index} column={column} value={getTruncated(value)} row={row} />;
-    }
+  if (column.key === 'speciesScientificNames') {
+    return <CellRenderer index={index} column={column} value={getTruncated(value)} row={row} />;
+  }
 
-    if (column.key === 'plotNames' && value) {
-      return <CellRenderer index={index} column={column} value={getTruncated([value])} row={row} />;
-    }
+  if (column.key === 'plotNames' && value) {
+    return <CellRenderer index={index} column={column} value={getTruncated([value])} row={row} />;
+  }
+
+  if (column.key === 'withdrawnDate') {
+    return <CellRenderer index={index} column={column} value={createLinkToNurseryWithdrawalDetail(value)} row={row} />;
+  }
 
-    if (column.key === 'withdrawnDate') {
+  if (column.key === 'hasReassignments') {
+    if (row.purpose === OUTPLANT && row.plotNames) {
       return (
-        <CellRenderer index={index} column={column} value={createLinkToNurseryWithdrawalDetail(value)} row={row} />
+        <>
+          <CellRenderer
+            index={index}
+            column={column}
+            row={row}
+            value={
+              <Button
+                id='reassign'
+                label={strings.REASSIGN}
+                onClick={rowClick}
+                size='small'
+                priority='secondary'
+                className={classes.text}
+                disabled={isTrue(row.hasReassignments)}
+              />
+            }
+          />
+        </>
       );
     }
+    return <CellRenderer index={index} column={column} row={row} value='' />;
+  }
 
-    if (column.key === 'totalWithdrawn') {
-      return <CellRenderer index={index} column={column} value={formatter.format(value as number)} row={row} />;
-    }
-
-    if (column.key === 'hasReassignments') {
-      if (row.purpose === OUTPLANT && row.plotNames) {
-        return (
-          <>
-            <CellRenderer
-              index={index}
-              column={column}
-              row={row}
-              value={
-                <Button
-                  id='reassign'
-                  label={strings.REASSIGN}
-                  onClick={rowClick}
-                  size='small'
-                  priority='secondary'
-                  className={classes.text}
-                  disabled={isTrue(row.hasReassignments)}
-                />
-              }
-            />
-          </>
-        );
-      }
-      return <CellRenderer index={index} column={column} row={row} value='' />;
-    }
-
-    return <CellRenderer {...props} />;
-  };
-};
-
-export default WithdrawalLogRenderer;
+  return <CellRenderer {...props} />;
+}
```

Comment:
All these changes are basically a revert of a previous PR that introduced client-side formatting of numeric data. Removed that since we will use formatted numbers from the BE.